sprintMission4_/src/middlewares/error.js

```diff
@@ -0,0 +1,15 @@
+import prisma from '../lib/prisma';
```

Comment:
ESM import 구문은 확장자가 명시되어야합니다.

// package.json:13
"type": "module"

```suggestion
import prisma from '../lib/prisma.js';
```